test/cli.test.js

```diff
@@ -151,3 +151,105 @@ test('validate command - invalid dataset', async t => {
 module.exports = {
   runcli
 }
+
+// QA tests [Cat: basic csv]
+
+test('cat command - basic behaviour', async t => {
+  const path_ = 'test/fixtures/test-data/files/csv/all-schema-types.csv'
+  const results = await runcli('cat', path_)
+  const stdout = results.stdout.split('\n')
+  t.true(stdout[3].includes('│ 1.0  │'))
+})
+
+test('cat command - remote csv file', async t => {
+  const url_ = 'https://raw.githubusercontent.com/frictionlessdata/test-data/master/files/csv/all-schema-types.csv'
```

Comment:
Just suggestion, we can support GitHub files as we did in `data-get`  data cat https://github.com/frictionlessdata/test-data/blob/master/files/csv/all-schema-types.csv?raw=true. But, it is different issue

Reply:
@Mikanebu I don't think we need, it was kind of experimental for get command